src/VirtualDevice.ts

```diff
@@ -17,10 +33,29 @@ export class VirtualDevice {
                         public locationLat?: string,
                         public locationLong?: string,
                         public conversationId?: string,
+                        public screenMode?: string,
+                        public client?: string,
                         ) {
         this.baseURL = process.env.VIRTUAL_DEVICE_BASE_URL
             ? process.env.VIRTUAL_DEVICE_BASE_URL
             : "https://virtual-device.bespoken.io";
+
+        if (arg0 === Object(arg0)) {
```

Comment:
why do we need this?

Reply:
it is because were are adding a lot of parameters to the constructor. So I added an option to pass an object instead as first parameter. This object has all the properties.